### PR TITLE
Logan/update docs

### PR DESCRIPTION
**Pull request by logan-markewich**

### CODE REVIEW

**Review by hatianzhang — APPROVED** [review]
i will remove baseline one

File: llama_parse/base.py [hunk 1]
```diff
@@ -22,7 +24,7 @@ class LlamaParse(BasePydanticReader):
 
     api_key: str = Field(default="", description="The API key for the LlamaParse API.")
     base_url: str = Field(
-        default="https://api.cloud.llamaindex.ai/api/parsing",
+        default=DEFAULT_BASE_URL,
```
Comment (Disiok) [inline llama_parse/base.py:27]: I think you need to append `api/parsing` in the httpx call now?

**Review by Disiok — CHANGES_REQUESTED** [review]
as comment

File: llama_parse/base.py (hunk 1 above)
Comment (logan-markewich) [reply llama_parse/base.py:27]: oh whoops lol

File: llama_parse/base.py [hunk 2]
```diff
@@ -51,6 +53,13 @@ def validate_api_key(cls, v: str) -> str:
             return api_key
         
         return v
+    
+    @validator("base_url", pre=True, always=True)
+    def validate_base_url(cls, v: str) -> str:
+        """Validate the base URL."""
+        if os.getenv("LLAMA_CLOUD_BASE_URL", DEFAULT_BASE_URL):
```
Comment (Disiok) [inline llama_parse/base.py:60]: I think this will always return True?

Comment (logan-markewich) [reply llama_parse/base.py:60]: ah good point, it will

**Pull request merged**